cryptovoxels/integration.js

```diff
@@ -229,17 +229,7 @@ function sendOnLoadMetric(space) {
     const spaceCounterEndpoint = API_BASE + `/api/v1/space/${space}`
     fetch(spaceCounterEndpoint, { method: 'PUT' });
 
-    fetch(
-      BEACON_GRAPHQL_URI,
-      { 
-        method: 'POST',
-        mode: 'no-cors',
-        headers: {
-          'Content-Type': 'application/json'
-        },
-        body: JSON.stringify(`mutation { increment(eventType: visits, spaceId: "${space}") { message } }`)
-      }
-    );
+    fetch(`https://forward.zesty.market/${NETWORK}/space/${SPACE}/visit`);
```

Comment:
Can we have these endpoints in a constant to make it easier to manage in the future?

Reply:
Yeah I can do that